src/content/developers/docs/transactions/index.md

```diff
@@ -120,7 +124,7 @@ Bob's account will be debited **-1.0042 ETH**
 
 Alice's account will be credited **+1.0 ETH**
 
-The miner processing the transaction will get **+0.0042 ETH**
+The base fee will be burned **-0.0042**
```

Comment:
```suggestion
The base fee will be burned **-0.0042211**
```

Reply:
I don't think this is correct. 

Base fee = 0.0042 ETH
Tip = 0.0000211 ETH
Total fee = 0.0042211 ETH